src/Application/FileProcessor/PhpFileProcessor.php

```diff
@@ -116,19 +115,6 @@ public function process(File $file, Configuration $configuration): array
         return $systemErrorsAndFileDiffs;
     }
 
-    public function supports(File $file, Configuration $configuration): bool
-    {
-        return true;
-    }
-
-    /**
-     * @return string[]
-     */
-    public function getSupportedFileExtensions(): array
-    {
-        return ['php'];
-    }
```

Comment:
This was actually a bug, as own `$rectorConfig->fileExtensions()` is ignored here